src/sql/workbench/common/editor/query/queryEditorInput.ts

```diff
@@ -284,6 +285,11 @@ export abstract class QueryEditorInput extends EditorInput implements IConnectab
 			fullTitle += ` (${profile.userName || profile.authenticationType})`;
 
 			fullTitle += additionalOptions;
+
+			if (info && info.serverConnectionId) {
+				// Add server info to query editor in case if it's available
+				fullTitle += ` ${localize('queryEditorInput.serverConnectionId', "Server Connection Process ID: {0}", info.serverConnectionId)}`
```

Comment:
Can we simplify it to how SSMS shows - also add it in the beginning of the title?